dipy/reconst/shm.py

```diff
@@ -426,39 +469,35 @@ def real_full_sh_basis(sh_order, theta, phi):
            Magn. Reson. Med. 2007;58:497-510.
 
     """
-    m, n = sph_harm_full_ind_list(sh_order)
-    phi = np.reshape(phi, [-1, 1])
-    theta = np.reshape(theta, [-1, 1])
-
-    real_sh = real_sph_harm(m, n, theta, phi)
-    return real_sh, m, n
+    deprecate_with_version('Function real_sym_sh_basis has been replaced \
```

Comment:
I recommend to : 
- use it as a decorator
- add from and until when

look [here](https://github.com/dipy/dipy/pull/2191/files#diff-3d2aafe93b8f41be9cf3521aa9d5d194R20-R24) as an example